### ISSUE TITLE
Existing answers are (mostly) junk Helm advice

### ISSUE THREAD

**Issue by demon** [pulumi/pulumi-ai#74]
Navigate to [the existing answers](https://www.pulumi.com/ai/answers). If you page through them, it's just row after row after row of "install X helm chart on AKS/EKS/GKE/Rancher/OpenShift/Oracle." 

There's very little difference between Helm chart installation on different k8s providers (most of the time). Additionally, all of the helm charts are in reverse alphabetical order, which makes me think someone spammed the heck out of the tool with a list of helm charts.

Basically: the existing answers are _useless_ as a source of inspiration. I don't know exactly what I'm asking for here...maybe a search box/filter options?

**Comment by AaronFriel** [comment]
Hey @demon, definitely hear you on needing to provide better search, sorting, and filtering. In the mean time, using `site:pulumi.com` in a search engine should provide good enough results. We'll keep this issue open to track improvements to Pulumi AI Answers.
